tripal_feature/includes/tripal_feature.gff_loader.inc

```diff
@@ -650,7 +650,12 @@ function tripal_feature_load_gff3($gff_file, $organism_id, $analysis_id,
         $strand = -1;
       }
       if (strcmp($phase, '.') == 0) {
-        $phase = '';
+        if ($type == 'CDS') {
```

Comment:
This looks like code from another issue.  If you remove it I will approve and merge

Reply:
Oh there was no other issue. It was a fix that I copied from the Tv3 code that was missing in Tv2.   I didn't mean for that to get into this PR, but given that's it's identical to Tv3 code can we leave it in. so it gets merged.  Here's the same code in Tv3:

https://github.com/tripal/tripal/blob/44ba26f4bd17b5f52f6c042d0e9e58543db190fd/tripal_chado/includes/TripalImporter/GFF3Importer.inc#L565-L573

Reply:
oh shoot im sorry, looks good then, lets merge.

Reply:
Oh, good call.  I'm glad you saw it.